src/components/Create/utils/formatFundingTarget.ts

```diff
@@ -0,0 +1,26 @@
+import { BigNumber } from '@ethersproject/bignumber'
+import { V2V3CurrencyOption } from 'models/v2v3/currencyOption'
+import { formatCurrencyAmount } from 'utils/formatCurrencyAmount'
+import { V2V3_CURRENCY_ETH } from 'utils/v2v3/currency'
+import { MAX_DISTRIBUTION_LIMIT } from 'utils/v2v3/math'
+
+export function formatFundingTarget({
+  distributionLimit,
+  distributionLimitCurrency = V2V3_CURRENCY_ETH.toString(),
+}: {
+  distributionLimit: string | undefined
+  distributionLimitCurrency: string | undefined
```

Comment:
**question**: would it make sense to

```suggestion
  distributionLimitCurrency: V2V3CurrencyOption | undefined
```

here?

Reply:
right now our uses of it prefer to use it as string - we could make it accept either and parse as needed though